service/share/autoyast-compat.json

```diff
@@ -68,6 +68,7 @@
           { "key": "description", "support": "no" },
           {
             "key": "bootproto",
+            "support": "no",
```

Comment:
@imobachgs Creating the schema did help finding the missing `support` here. Yay! Is `no` right?